### PR TITLE
Test enhancement

### PR DESCRIPTION
**Pull request by peter279k**
Reopen this PR from [here](https://github.com/mtdowling/cron-expression/pull/167).

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -3,6 +3,7 @@ language: php
 php:
   - 7.0
   - 7.1
+  - 7.2
```
[the remark this answers is not in the document]
Comment (peter279k) [reply .travis.yml]: Hi @simPod, It makes sense.
I will add the `php-nightly` and `hhvm` tests in Travis CI build and allow them to be failed 👍 .

File: tests/Cron/DayOfMonthFieldTest.php [hunk 2]
```diff
@@ -34,6 +34,57 @@ public function testChecksIfSatisfied()
...
+        $this->assertTrue($f->validate('2,12'));
+    }
+
+    /**
+     * @covers \Cron\DayOfMonthField::isSatisfiedBy
+     */
+    public function testIsSatipsfiedByOnLValue()
+    {
+        $f = new DayOfMonthField();
+        $this->assertFalse($f->isSatisfiedBy(new DateTime, 'L'));
```
Comment (phansys) [inline tests/Cron/DayOfMonthFieldTest.php]: Please, respect existing CS by using parentheses when instantiating classes regardless of the number of arguments its constructor has.

File: composer.json [hunk 3]
```diff
@@ -20,7 +20,7 @@
         "php": ">=7.0.0"
     },
     "require-dev": {
-        "phpunit/phpunit": "~6.4"
+        "phpunit/phpunit": "^6.5"
```
[the remark this answers is not in the document]
Comment (peter279k) [reply composer.json]: Using `^6.5` definition can detect the `^6.5+` version and this definition can be more clear than defining `~6.4` version.

**Comment by peter279k** [comment]
Hi @phansys, thank you for your review. I appreciate your help.

I've accept request changes and push the latest commit.

Please review them. Thanks.

**Pull request closed without merging**